docker-compose.yml

```diff
@@ -123,14 +138,15 @@ services:
       - redis
       - postgres
     ports:
-      - 8443:8443
+      - 443:8443
```

Comment:
I must have something configured differently because this doesn't work for me.
With this change, the container never starts for me locally, it keeps waiting at `Attaching to service`.

Reply:
I changed my `secrets/packit/dev/packit-service.yaml`
```yaml
#server_name: stg.packit.dev
#server_name: service.localhost:8443
server_name: service.localhost
#dashboard_server_name: dashboard.localhost
#dashboard_url: https://dashboard.stg.packit.dev
dashboard_url: https://dashboard.localhost
```

I tried to use port 443 in dashboard but I can not remember why I did not manage it. But with the above changes I can query both the API and the dashboard.

Reply:
receck

Reply:
hmm, still the same, i.e. waiting  for `Attaching to service` :crying_cat_face:

Reply:
Here the logs from my docker-compose run: https://gist.github.com/majamassarini/8a089630519153dcd7a019a1ec295501

maybe you can spot some difference?

Reply:
OK, tried again and this time podman gave me a reason why it doesn't work

```
Attaching to service
Error response from daemon: rootlessport cannot expose privileged port 443,
you can add 'net.ipv4.ip_unprivileged_port_start=443' to /etc/sysctl.conf (currently 1024),
or choose a larger port number (>= 1024): listen tcp 0.0.0.0:443: bind: permission denied
```

and
`echo "net.ipv4.ip_unprivileged_port_start=443" > /etc/sysctl.d/docker-compose.conf; sysctl --system`
makes it work, indeed, but I'm not sure that's a way I want to follow.

Could we use a port higher than 1024 (e.g. 1443 or 9443) for either the service or (preferably) the dashboard?

Reply:
Oh you are right, in the past I had changed the `net.ipv4.ip_unprivileged_port_start` value in my laptop,so I did not realize this configuration was a problem!
Yes, I can try to use port 9443 for dashboard.